src/core/js/app.js

```diff
@@ -203,6 +204,10 @@ require([
         }
     }
 
+    function removeInitialLoadingMessage() {
```

Comment:
`$('#initial-loading').remove();` seems clear enough to me but I'd be interested to hear what others thing